contracts/DaiHrd.sol

```diff
@@ -569,28 +562,26 @@ contract DaiHrd is ERC777, MakerFunctions {
 	}
 
 	// Utility Functions
-	function convertAttodaiToAttodaiHrd(uint256 attodai, uint256 rontodaiPerPot ) internal pure returns (uint256 attodaiHrd) {
+	function calculatedChi() public view returns (uint256 rontodaiPerPot) {
+		// mirrors Maker's calculation: rmul(rpow(dsr, now - rho, ONE), chi);
+		return safeMul27(rpow(pot.dsr(), now - pot.rho(), 10 ** 27), pot.chi());
+	}
+
+	function convertAttodaiToAttodaiHrd(uint256 attodai, uint256 rontodaiPerPot ) private pure returns (uint256 attodaiHrd) {
```

Comment:
I made all `internal` functions `private`.  I don't believe this matters, but I think it makes auditing _slightly_ easier (lower cognitive load on auditor).

Reply:
More restrictive 👍